server/jetstream_cluster_test.go

```diff
@@ -4563,6 +4563,85 @@ func TestJetStreamClusterSuperClusterMetaPlacement(t *testing.T) {
 	}
 }
 
+func TestJetStreamUniquePlacementTag(t *testing.T) {
+	tmlp := `
+		listen: 127.0.0.1:-1
+		server_name: %s
+		jetstream: {max_mem_store: 256MB, max_file_store: 2GB, store_dir: '%s', limits: {unique_placement_prefix: az}}
```

Comment:
This feels like its not a descendant of limits imo. Just inline with max_file_store etc. But could be wrong.